gulpfile.js

```diff
@@ -0,0 +1,44 @@
+var gulp                = require('gulp'),
+    livereload          = require('gulp-livereload'),
+    nodemon             = require('gulp-nodemon'),
+    shell               = require('gulp-shell'),
+    chalk               = require('chalk'),
+    paths,
+    nodemonServerInit;
+
+paths = {
+    server: 'core/server/',
+    client: 'core/client/'
+};
+
+nodemonServerInit = function () {
+    livereload.listen();
+
+    return nodemon({
+        script: 'index.js',
+        ext: 'js,json',
+        watch: [
+            'core/index.js',
+            paths.server + '**/*.js',
+            'core/built/assets/*.js'
+        ],
+        ignore: [
+            'core/client/*',
+            'core/server/test/*'
+        ]
+    }).on('restart', function () {
+        console.info(chalk.cyan('Restarting Ghost due to changes...'));
+        gulp.src('index.js')
+            .pipe(livereload());
+    });
+};
+
+// Starting Ember for development
+gulp.task('admin', shell.task(['cd ' + paths.client + ' && ember build --watch']));
+
+gulp.task('serve', function () {
```

Comment:
Hannah's suggestion makes sense when you consider the `admin` task being for internal use rather than anything a user would actually run.

Being able to start `ember serve` via `gulp admin` task only really makes sense once ember is in control of it's own `index.html` and if you can pass in all the options that `ember serve` allows, e.g. `gulp admin --proxy http://127.0.0.1:3000` at which point I'm not sure what benefit we have over pointing ember devs at the standard ember-cli docs?